{{cookiecutter.project_name}}/docs/pages/template/testing.rst

```diff
@@ -108,4 +108,4 @@ for more information.
 We also use `django-stubs <https://github.com/typeddjango/django-stubs>`_
 to type ``django`` internals.
 This package is optional and can be removed,
-if you don't what to type your ``django`` for some reason.
+if you don't want to type your ``django`` for some reason.
```

Comment:
This change is unrelated, please remove it

Reply:
I'm sorry, how can I remove unrelated commit?

Reply:
It is easier to open a new PR in this case 🙂